locales/ar/json.json

```diff
@@ -906,8 +906,8 @@
     "The resource was attached!": "تم إرفاق المصدر!",
     "The resource was prevented from being saved!": "تم منع عملية حفظ المصدر!",
     "The resource was updated!": "تم تحديث المصدر!",
-    "The response is not a streamed response.": "The response is not a streamed response.",
-    "The response is not a view.": "The response is not a view.",
+    "The response is not a streamed response.": "الاستجابة ليست استجابة متدفقة",
```

Comment:
Is a full stop at the end of the sentence needed here or is it not necessary?

Reply:
It's not necessary but I forgot it. I'll add it to match the original.